bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -86,6 +87,15 @@ impl fmt::Display for PackedLockTime {
     }
 }
 
+impl FromHexStr for PackedLockTime {
+    fn from_hex_str_no_prefix<S: AsRef<str>>(s: S) -> Result<Self, FromHexError>
+        where Self: Sized
+    {
+        let sequence = u32::from_str_radix(s.as_ref(), 16).map_err(|e| FromHexError::ParseHex(e.to_string()))?;
```

Comment:
FYI `from_str_radix` may be slow but I didn't measure. Totally OK for now and someone can file a PR later if better perf is needed.

Reply:
For sure, this seemed to be the easiest and most readable way to do this that I found without adding more dependencies